examples/serverless/main.tf

```diff
@@ -182,8 +182,9 @@ module "aurora_postgresql_v2" {
   enable_http_endpoint = true
 
   serverlessv2_scaling_configuration = {
-    min_capacity = 2
-    max_capacity = 10
+    min_capacity             = 0
+    max_capacity             = 10
+    seconds_until_auto_pause = 3600
```

Comment:
any reason it was set for 1 hour?
console defaults to 5 mins

Reply:
This is just an example. Real values should be decided by users individually.

Reply:
yes using this already, thanks, was just reffering what was the default in the console when adding this setting ;)